PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/lib/appservice/AppServiceConfig.java

```diff
@@ -38,13 +39,13 @@
 @Data
 @SuperBuilder
 public class AppServiceConfig {
-    public static final Platform DEFAULT_PLATFORM = Platform.Linux.JAVA8_TOMCAT9;
     public static final PricingTier DEFAULT_PRICING_TIER = PricingTier.BASIC_B2;
-
+    @Builder.Default
+    private MonitorConfig monitorConfig = MonitorConfig.builder().build();
     private String name;
     private Path application;
     @Builder.Default
-    private Platform platform = DEFAULT_PLATFORM;
+    private Platform platform = WebAppConfig.DEFAULT_PLATFORM;
```

Comment:
move `DEFAULT_PLATFORM` to `AppServiceConfig`?

Reply:
As Function and Web App has different default platform, moving the default value to correspond config class